homeassistant/components/axis/camera.py

```diff
@@ -73,7 +73,7 @@ def supported_features(self):
         """Return supported features."""
         return SUPPORT_STREAM
 
-    async def stream_source(self):
+    def stream_source(self):
```

Comment:
Any method that does not I/O, like this one, should remain in async as it is a lot faster.

Reply:
Oh yeah. Makes sense. I'll revisit this.